README.md

```diff
@@ -93,12 +93,28 @@ The `wifi` section below includes the following entries:
 {
   "wifi": {
     "ssid": "MY_SSID",
-    "password": "MY_PASSWORD"
+    "password": "MY_PASSWORD",
+    "eap": {
+      "method": null,
+      "identity": null,
+      "client_cert": null,
+      "client_key": null,
+      "server_cert": null,
+      "username": null,
+      "password": null
+    }
   }
 }
 ```
 * `ssid` - The WiFi SSID the ESP32 should connect to
 * `password` - The security password for the above network
+* `eap` - WPA-Enterprise configuration (for enterprise networks only)
+  * `method` - `TLS`, `PEAP` or `TTLS`
+  * `identity` - The EAP identity
+  * `ca_cert`, `client_cert`, `client_key` - Full path names, including a
+    leading slash (/), of the certificate/key file (in respectively PEM, CRT and
+    KEY format) stored under the data folder.
```

Comment:
Correct me if I'm wrong, but aren't they all PEM format?

Reply:
You are right, crt (e.g. [wpa2_client.crt](https://github.com/espressif/esp-idf/blob/release/v3.2/examples/wifi/wpa2_enterprise/main/wpa2_client.crt)) and key [wpa2_client.key](https://github.com/espressif/esp-idf/blob/release/v3.2/examples/wifi/wpa2_enterprise/main/wpa2_client.key) files are in PEM format, just with [different extension](https://serverfault.com/a/9717/405370). I'll fix the README.